### PR TITLE
[IGOWEB-1521] Add endpoint for dmp samples retrieval

### PR DESCRIPTION
**Pull request by imprezka88**

### CODE REVIEW

File: src/main/java/org/mkscc/igo/pi/external/rest/ServiceExternalSampleRepository.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import org.mskcc.domain.external.ExternalSample;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework. beans.factory.annotation.Value;
```
Comment (lvncnt) [inline src/main/java/org/mkscc/igo/pi/external/rest/ServiceExternalSampleRepository.java]: is that a space there?

File: src/main/java/org/mkscc/igo/pi/dmptoigo/dmp/DmpToIgoController.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    private DMPSamplesGateway dmpSamplesGateway;
+    private Notificator notificator;
+
+    @Autowired
+    public DmpToIgoController(DMPSamplesGateway dmpSamplesGateway, Notificator notificator) {
+        this.dmpSamplesGateway = dmpSamplesGateway;
+        this.notificator = notificator;
+    }
+
+    @RequestMapping("/dmp-to-igo")
```
Comment (lvncnt) [inline src/main/java/org/mkscc/igo/pi/dmptoigo/dmp/DmpToIgoController.java]: should it be a POST/PUT request?

File: src/main/java/org/mkscc/igo/pi/external/rest/ServiceExternalSampleRepository.java (hunk 1 above)
Comment (imprezka88) [reply src/main/java/org/mkscc/igo/pi/external/rest/ServiceExternalSampleRepository.java]: removed

File: src/main/java/org/mkscc/igo/pi/dmptoigo/dmp/DmpToIgoController.java (hunk 2 above)
Comment (imprezka88) [reply src/main/java/org/mkscc/igo/pi/dmptoigo/dmp/DmpToIgoController.java]: changed

**Review by lvncnt — APPROVED** [review]

**Pull request merged**